### DISCUSSION TITLE
Error Logging in FastHttpUser

### DISCUSSION THREAD

**Discussion by swaalt** [locustio/locust#2936] [General]
Hi, I am using FastHttpUser and when I receive a 404 the fail is registered like this, this happens because the geventhttpclient saves in the exception the badstatuscode text the url of the request I made, which is dynamic and appears separated by each one,
I need that either I take the “name” or simply do not place anything, just place the badstatuscode 404, can this be addressed with the locust as it is now or do I have to modify something?

![fails](https://github.com/user-attachments/assets/92b13451-f29c-4c8c-a42c-7f1678380deb)

**Comment by cyberw** [comment]
Hi! I think we should solve this in Locust and have created a ticket based on this.

As a workaround (because I'm not sure anyone will get to this soon) you can overwrite the exception message using something like this:

        with self.client.get("/", catch_response=True) as response:
            if response.status_code == 404:
                response.failure("404")

**Reply by swaalt** [reply]
thank you very much @cyberw !
